test/ClangImporter/Inputs/custom-modules/module.modulemap

```diff
@@ -113,8 +113,8 @@ module ImportAsMember {
   }
 }
 
-// FIXME: This probably ought to be in a module_private.map, but that causes
-// hundreds of clang warnings.
+// FIXME: This probably ought to be in a module.private.modulemap, but that
+// causes hundreds of clang warnings.
```

Comment:
What's this actually referring to?

Reply:
The comment was added in #36198 a few years ago, but I didn't remember the exact details. I made the suggested change locally and got a ton of these:

```
/Volumes/DocumentsHD/Code/open-swift-alt/swift/test/ClangImporter/Inputs/custom-modules/module.private.modulemap:3:8: error: expected canonical name for private module 'ImportAsMember_Private'
 1 | // FIXME: This probably ought to be in a module.private.modulemap, but that
 2 | // causes hundreds of clang warnings.
 3 | module ImportAsMember_Private {
   |        |- error: expected canonical name for private module 'ImportAsMember_Private'
   |        `- note: rename 'ImportAsMember_Private' to ensure it can be found by name
 4 |   export *
 5 | 
```

This warning is intended to be emitted if module.private.modulemap has a module named, say, `ImportAsMemberPrivate` and clang thinks it ought to be called `ImportAsMember_Private`. I think perhaps it's confused because module.modulemap contains many modules (not just the public `ImportAsPrivate`) and `ImportAsMember_Private` doesn't match the expected name for those other modules.